src/constants.ts

```diff
@@ -8,7 +8,7 @@ import * as nls from 'vscode-nls';
 
 const localize = nls.loadMessageBundle();
 
-export const serviceName = localize('extension.serviceName', 'pgSQLToolsService');
+export const serviceName = localize('extension.serviceName', 'ossdbToolsService');
 export const providerId = localize('extension.providerId', 'PGSQL');
 export const serviceCrashMessage = localize('extension.serviceCrashMessage', 'PG SQL Tools Service component exited unexpectedly. Please restart Azure Data Studio.');
```

Comment:
Change the serviceCrashMessage text to "Oss DB Tools Service component exited …..."